experimental/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to experimental packages in this project will be documented
 
 ### :rocket: (Enhancement)
 
+* feat: improve http span name [#3603](https://github.com/open-telemetry/opentelemetry-js/pull/3603) @Flarna
```

Comment:
I think this is considered a breaking change since any queries by span name will break